lib/librarian/puppet/util.rb

```diff
@@ -26,7 +26,21 @@ def rsync?
       # If the rsync configuration parameter is set, use rsync instead of FileUtils
       def cp_r(src, dest)
         if rsync?
-          Rsync.run(File.join(src, "/"), dest, ['-avz', '--delete'])
+          if Gem.win_platform?
+            src_clean = "#{src}".gsub(/^([a-z])\:/i,'/cygdrive/\1')
+            dest_clean = "#{dest}".gsub(/^([a-z])\:/i,'/cygdrive/\1')
+          else
+            src_clean = src
+            dest_clean = dest
```

Comment:
When running on windows with a "cygwin powered rsync", you must be running rsync with 

```
rsync /cygdrive/c/source/  /cygdrive/c/dest/
```

If you run with rsync c:/source/  c:/dest/, rsync complains that not both sides can be local (path for remote in rsync is HOST:/path)

This codes translates c: or C: to /cygdrive/c/ in the path to allow rsync to work.